### PR TITLE
Add tax for post source

### PR DESCRIPTION
**Pull request by varffs**

### CODE REVIEW

File: lib/post-types.php [hunk 1]
```diff
@@ -53,3 +53,44 @@ function programacion_post_type() {
...
+		'separate_items_with_commas' => __( 'Separate items with commas', 'text_domain' ),
+		'add_or_remove_items'        => __( 'Add or remove items', 'text_domain' ),
+		'choose_from_most_used'      => __( 'Choose from the most used', 'text_domain' ),
+		'popular_items'              => __( 'Popular Items', 'text_domain' ),
+		'search_items'               => __( 'Search Items', 'text_domain' ),
+		'not_found'                  => __( 'Not Found', 'text_domain' ),
+		'no_terms'                   => __( 'No items', 'text_domain' ),
+		'items_list'                 => __( 'Items list', 'text_domain' ),
+		'items_list_navigation'      => __( 'Items list navigation', 'text_domain' ),
+	);
```
Comment (mikeyrayvon) [inline lib/post-types.php:83]: `text_domain`? hows that work? where did u get that from?

Comment (varffs) [inline lib/post-types.php:83]: doesnt do anything just comes default from the wp generator i use

Comment (mikeyrayvon) [inline lib/post-types.php:83]: maybe we should switch the copy for español? @mixcas ?

Comment (mixcas) [reply lib/post-types.php:83]: are we using WP in spanish or english right now?

Comment (varffs) [reply lib/post-types.php:83]: depends on the user their pref if q-translate is installed

**Pull request merged**